src/hooks/useGetRewards.ts

```diff
@@ -3,14 +3,19 @@ import { useGetQuest } from './useGetQuest'
 import { getDecimalNumberFromAmount, Quest } from '@hyperplay/utils'
 import { getRewardCategory } from '../helpers/getRewardCategory'
 import { useTranslation } from 'react-i18next'
-import { QuestReward } from '@hyperplay/ui'
+import { UseGetRewardsData } from '@/types/quests'
 
-export function useGetRewards(
-  questId: number | null,
-  getQuest: (questId: number) => Promise<Quest>,
-  getExternalTaskCredits: (rewardId: string) => Promise<string>,
+export function useGetRewards({
+  questId,
+  getQuest,
+  getExternalTaskCredits,
+  logError
+}: {
+  questId: number | null
+  getQuest: (questId: number) => Promise<Quest>
+  getExternalTaskCredits: (rewardId: string) => Promise<string>
```

Comment:
Could get these types from `QuestWrapperContextValue` to prevent redefining them.